webapp/components/admin_console/users_and_teams_settings.jsx

```diff
@@ -188,6 +190,28 @@ export default class UsersAndTeamsSettings extends AdminSettings {
                     value={this.state.restrictDirectMessage}
                     onChange={this.handleChange}
                 />
+                <DropdownSetting
+                    id='teammateDisplay'
+                    values={[
+                        {value: Constants.TEAMMATE_DISPLAY.SHOW_USERNAME, text: Utils.localizeMessage('admin.team.showUsername', 'Show username')},
+                        {value: Constants.TEAMMATE_DISPLAY.SHOW_NICKNAME_FULLNAME, text: Utils.localizeMessage('admin.team.showNickname', 'Show nickname if one exists, otherwise show first and last name')},
+                        {value: Constants.TEAMMATE_DISPLAY.SHOW_FULLNAME, text: Utils.localizeMessage('admin.team.showFullname', 'Show first and last name (default)')}
+                    ]}
+                    label={
+                        <FormattedMessage
+                            id='admin.team.teammateDisplay'
+                            defaultMessage='Teammate Name Display:'
+                        />
+                    }
+                    helpText={
+                        <FormattedHTMLMessage
+                            id='admin.team.teammateDisplayDesc'
+                            defaultMessage="Set how to display user's names in posts and the Direct Messages list."
```

Comment:
Grammar mistake, I think `user's` should be `users'`, @jasonblais (you opened the ticket, sorry if you're not the PM on this)